src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -139,6 +146,22 @@ def _getResultsFileName(self, toilPath):
         """
         return os.path.join(toilPath, "results.txt")
 
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
+        """
+        raise NotImplementedError('Abstract method: supportsWorkerCleanup')
```

Comment:
The message argument to the exception constructor is redundant (the stacktrace will make it obvious which method is affected) and bound to get inconsistent with the actual name of the method.